pysdsl/types/intvector.hpp

```diff
@@ -169,88 +166,81 @@ inline auto add_int_class(py::module& m, py::dict& dict, KEY_T key,
 }
 
 
-inline auto add_int_vectors(py::module& m)
-{
-    py::dict int_vectors_dict;
+struct AddIntVectorFunctor {
+    py::module& m;
+    py::dict& int_vectors_dict;
 
-    m.attr("int_vector") = int_vectors_dict;
+    AddIntVectorFunctor(py::module& m, py::dict& int_vectors_dict)
+        : m(m), int_vectors_dict(int_vectors_dict) {}
```

Comment:
looks like we only use refs here, so probably `constexpr` and `noexcept` constructor can be used.